build-art.js

```diff
@@ -0,0 +1,95 @@
+const fs = require('fs');
+const sharp = require('sharp');
+
+console.log('Sharp Versions:');
+console.log(sharp.versions);
+console.log(`Sharp Concurrency: ${sharp.concurrency()}`);
+
+const imagemin = require('imagemin');
+const imageminPngquant = require('imagemin-pngquant');
+
+/**
+ * Apply a sharp operation to all of the pictures in a directory, and write
+ * the output to a different directory.
+ *
+ * @param dirPath WHere to find the input pictures.
+ *
+ * @param outputDirPath Where to put the output pictures. (This directory will
+ *   be created if it does not already exist.)
+ *
+ * @param sharpOperations A function that takes a path to an image file,
+ *   applies some operations to it, and returns a Sharp object.
+ */
+async function processDirectory(dirPath, outputDirPath, sharpOperations) {
+  console.log(`Processing ${dirPath} to ${outputDirPath}`);
+  await fs.promises.mkdir(outputDirPath, { recursive: true });
+
+  const dir = await fs.promises.opendir(dirPath);
+
+  for await (const dirEntry of dir) {
+    if (dirEntry.isFile()) {
+      const filePath = `${dirPath}/${dirEntry.name}`;
+      const outputFilePath = `${outputDirPath}/${dirEntry.name}`;
+      const info = await sharpOperations(filePath).toFile(outputFilePath);
+      console.log(`Processed ${filePath} to ${outputFilePath} (${info.size} bytes)`);
+    }
+  }
+  console.log(`Done Sharp processing ${dirPath} to ${outputDirPath}`);
+  await imagemin([`${outputDirPath}/*.png`], {
+    destination: outputDirPath,
+    plugins: [
+        imageminPngquant({
+            quality: [0.3, 0.8]
+        })
+    ]
```

Comment:
```suggestion
    plugins: [
      imageminPngquant({
        quality: [0.3, 0.8]
      })
    ]
```
Just a quick indentation thing here